docs/api/commands/update-page-title.md

```diff
@@ -0,0 +1,17 @@
+# Update page title
+
+If you want to update the page title, you can use `me.update_page_title` which will
```

Comment:
I would name this `set_page_title` b/c:
1. consistency with our existing APIs, e.g. [set_theme_mode/density](https://github.com/google/mesop/blob/1f34f5669ba2ab064d91a6e11e1da5848bf92077/mesop/features/theme.py#L10)
2. in the case where the user hasn't set a page title (e.g. `me.page(` doesn't have the page arg), I think it's clearer calling this `set_page_title` rather than `update_page_title` since there wasn't really a page title to begin with (besides the default page title the framework sets)

Reply:
Great point. Updated.